### ISSUE TITLE
Autoscrolling Changes Broke "Castle of the Penguin King" from Super Talking Time Bros. 2

### ISSUE THREAD

**Issue by DerpyBubblez** [Wohlstand/TheXTech#65]
Expected behavior: https://youtu.be/e8_cvLOA_Qs?list=PL10AC6F2FAB3E1858&t=227

What happens: https://cdn.discordapp.com/attachments/215661302692052992/841088782114619393/Scr_2021-05-09_18-05-03.gif

**Comment by Wohlstand** [comment]
One another level to use the compat.ini

**Comment by DerpyBubblez** [comment]
Is there already an option for that, or do I need to wait?

**Comment by Wohlstand** [comment]
Yes, for EVERY behavior fix I do the compat.ini option to allow tune the compatibility cases. I'm on phone as I woke up just now, don't remind exact name, but you can easily find it on the Wiki at the bottom.

**Comment by Wohlstand** [comment]
Btw, the case of checkpoints was a one only that hadn't the "compat.ini" flag because it's one of earliest fixes I done.

**Comment by Wohlstand** [comment]
BTW, feel free to report any incompatibility cases here: #54, as soon I do plan to make the global compat list to enforce the compatibility for certain levels and episodes automatically, the same as GZDoom does
